test/components/notice_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'component_test_helper'
+
+class NoticeTest < ComponentTestCase
+  def component_name
+    "notice"
+  end
+
+  test "fails to render when no parameters are given" do
```

Comment:
We need to figure out what do with these tests, ideally we'd only have the integration test and that'd cover all options.